.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -282,7 +283,7 @@ jobs:
               -DEnzyme_DIR=$GITHUB_WORKSPACE/enzyme-build \
               -DENZYME_SRC_DIR=$GITHUB_WORKSPACE/mlir/Enzyme \
               -DLLVM_ENABLE_ZLIB=OFF \
-              -DLLVM_ENABLE_ZSTD=OFF
+              -DLLVM_ENABLE_ZSTD=FORCE_ON
```

Comment:
Based on [the LLVM docs](https://llvm.org/docs/CMake.html), the `FORCE_ON` value forces to build with `zstd` and raises an error if zstd is not found! We don't necessarily need this in our Makefile as it can be optional in the dev mode.

Reply:
Nice, good to know :)